lte/gateway/c/session_manager/StatsPoller.cpp

```diff
@@ -13,17 +13,27 @@
 #include <stdint.h>
 #include <atomic>
 #include <memory>
+#include <vector>
 #include "StatsPoller.h"
-#define COOKIE 0
-#define COOKIE_MASK 0
+#define OVS_COOKIE_MATCH_ALL 0xffffffff
 
 namespace magma {
 
 void StatsPoller::start_loop(
     std::shared_ptr<magma::LocalEnforcer> local_enforcer,
     uint32_t loop_interval_seconds) {
+  // for now poll all shards by passing in an empty vector
```

Comment:
This comment is a bit confusing maybe

// When no shard_ids are available poll all UEs